code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -124,3 +124,15 @@
 		icon_state = "biofoam1"
 	else
 		icon_state = "biofoam0"
+
+/obj/item/weapon/reagent_containers/hypospray/autoinjector/paralytic_injector
```

Comment:
if you can, it would be nice if this had an alternate sprite and inhand because it *is* a weapon